kube-client/src/config/mod.rs

```diff
@@ -128,9 +128,25 @@ pub struct Config {
     pub default_namespace: String,
     /// The configured root certificate
     pub root_cert: Option<Vec<Vec<u8>>>,
+    /// Set the timeout for connecting to the Kubernetes API.
+    ///
+    /// A value of `None` means no timeout
+    pub connect_timeout: Option<std::time::Duration>,
+    /// Set the timeout for the Kubernetes API response.
+    ///
+    /// A value of `None` means no timeout
+    pub read_timeout: Option<std::time::Duration>,
+    /// Set the timeout for the Kubernetes API request.
+    ///
+    /// A value of `None` means no timeout
+    pub write_timeout: Option<std::time::Duration>,
     /// Timeout for calls to the Kubernetes API.
     ///
     /// A value of `None` means no timeout
+    #[deprecated(
+        since = "0.75.0",
```

Comment:
didn't know what version to put here, should it be the current or next one?

Reply:
The next one is correct. I would add the time frame for it as well (+3 version);

I.e. `This member will be removed in 0.78.0.`

Reply:
Done